tests/Makefile

```diff
@@ -28,7 +28,7 @@ test_integration: venv/integration
 	    echo "    make $@ TEST_hosts=hosts"; \
 	    false; \
 	fi
-	venv/integration/bin/ansible-playbook -vvv -i "$(TEST_HOSTS)" integration/test.yml
+	ANSIBLE_HOST_KEY_CHECKING=False venv/integration/bin/ansible-playbook -vvv -i "$(TEST_HOSTS)" integration/test.yml
```

Comment:
I meant that this should be added this to the environment of CircleCI, since it's only needed there.  (Sorry, I should have just said so.)